src/common/options/global.yaml.in

```diff
@@ -5296,6 +5296,19 @@ options:
   desc: Remove allocation info from RocksDB and store the info in a new allocation file
   default: true
   with_legacy: true
+- name: bluestore_allocation_recovery_threads
+  type: uint
+  level: basic
+  desc: Amount of threads for allocation recovery after OSD crash.
+  long_desc: The optimal value varies. For NVMe DBs may be as large as 8.
```

Comment:
(add the 'after an OSD crash' here, in the long description.
And - possibly after the next round of fixes - I'd suggest involving the tech writers)